### ISSUE TITLE
FITSIO with Julia 1.0

### ISSUE THREAD

**Issue by hbouy** [JuliaAstro/FITSIO.jl#102]
Is the current version v0.11.0 of FITSIO supposed to work under Julia 1.0?  
I get the following error when starting FITSIO for the first time:

[ Info: Precompiling FITSIO [525bcba6-941b-5504-bd06-fd0dc1a4d2eb]
ERROR: LoadError: syntax: extra token "ImageHDU" after end of expression
Stacktrace:
 [1] include at ./boot.jl:317 [inlined]
 [2] include_relative(::Module, ::String) at ./loading.jl:1038
 [3] include(::Module, ::String) at ./sysimg.jl:29
 [4] top-level scope at none:2
in expression starting at /Users/Bob/.julia/packages/FITSIO/WueGk/src/FITSIO.jl:54

**Comment by giordano** [comment]
No, unfortunately it's not supposed to work with Julia 1.0 yet. There is a PR to fix this: #101
